### PR TITLE
(maint) Audit acceptance tests

### PR DESCRIPTION
**Pull request by johnduarte**
This commit adds beaker audit test tags for risk mitigation and layer
assessment.

### CODE REVIEW

File: acceptance/tests/proxy_connection.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@
 require 'puppet/acceptance/environment_utils.rb'
 
 test_name 'Connect via proxy' do
+
+  tag 'audit:high',      #
```
Comment (donoghuc) [inline acceptance/tests/proxy_connection.rb:7]: Not super important. Also spinning up proxy is slowish...

**Comment by donoghuc** [comment]
I think i should add a high priority tag to t https://github.com/puppetlabs/pxp-agent/pull/786/files#diff-c714da86f83b9fbf56eac941fb01367cR4

File: acceptance/tests/proxy_connection.rb (hunk 1 above)
Comment (johnduarte) [reply acceptance/tests/proxy_connection.rb]: Thanks @donoghuc I have changed the risk tage for this test to a 'medium'.

**Comment by donoghuc** [comment]
Would running with "jenkins please test this on..." on a couple targets be helpful in any way to verify this is doing what you want? https://github.com/puppetlabs/pxp-agent/pull/753#issuecomment-498402376

**Comment by johnduarte** [comment]
jenkins please test this on redhat7-64a,windows2016-64a

File: acceptance/tests/pxp_agent_version.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 test_name 'C93805 - pxp-agent - Versioning test'
```
Comment (McdonaldSeanp) [inline acceptance/tests/pxp_agent_version.rb:4]: @johnduarte you can just kill this one

**Comment by johnduarte** [comment]
Tests not tagged with 'audit:high' are skipped as expected.

File: acceptance/tests/pxp_module_bolt_task/task_purged_from_cache.rb [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 test_name 'remove old task from pxp-agent cache' do
```
Comment (McdonaldSeanp) [inline acceptance/tests/pxp_module_bolt_task/task_purged_from_cache.rb:4]: @johnduarte This one can be killed.

This functionality is covered in unit

File: acceptance/tests/pxp_agent_version.rb (hunk 2 above)
Comment (johnduarte) [reply acceptance/tests/pxp_agent_version.rb]: @McdonaldSeanp I have added a commit to delete this test.  Thanks.

File: acceptance/tests/pxp_module_bolt_task/task_purged_from_cache.rb (hunk 3 above)
Comment (johnduarte) [reply acceptance/tests/pxp_module_bolt_task/task_purged_from_cache.rb]: @McdonaldSeanp this has also been deleted.  Thanks.

**Comment by McdonaldSeanp** [comment]
@johnduarte feel free to merge this whenever you think it's ready.

We should find some time to sit down and maybe risk assess some of the module tests. My gut feeling is there's some tests in there that can be squashed or killed.

**Pull request merged**